testgrid/config/config.yaml

```diff
@@ -1456,6 +1458,8 @@ dashboards:
     test_group_name: kubernetes-e2e-gke-updown
   - name: gci-gke-updown
     test_group_name: kubernetes-e2e-gci-gke-updown
+  - name: gke-gci-ci-master
```

Comment:
The ci-kubernetes-e2e-gke-gci-ci-master tests a canary version of the gci image, so it seems like it fits on that tab rather than the catch-all GKE one. @mtaufen can make the call.